### DISCUSSION TITLE
Silly question concerning wire length and macro&global placement

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD#5808] [General]
It looks like wire length would be a good optimization target for global placement and macro placement, looking at the screenshot from megaboom below:

Silly questions:

Is wire length driven macro/global placement a thing?

Is it somehow covered by routing + timing driven global placement?

More than 50% of this path is non-logic delay:

![image](https://github.com/user-attachments/assets/c891675a-5cca-4bbd-8c7a-630e752d0cb5)

**Comment by maliberty** [comment]
Write length minimization is a goal
